Student/Student/Submissions/SubmissionRubric/RubricViewController.swift

```diff
@@ -52,6 +55,31 @@ class RubricViewController: UIViewController {
 }
 
 extension RubricViewController: UICollectionViewDataSource, UICollectionViewDelegate, UICollectionViewDelegateFlowLayout {
+
+    func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, referenceSizeForHeaderInSection section: Int) -> CGSize {
+        if presenter.assignments.first?.useRubricForGrading ?? false {
+            return CGSize(width: collectionView.frame.width, height: 156)
+        } else {
+            return CGSize(width: collectionView.frame.width, height: 0)
+        }
+    }
+
+    func collectionView(_ collectionView: UICollectionView, viewForSupplementaryElementOfKind kind: String, at indexPath: IndexPath) -> UICollectionReusableView {
+        switch kind {
+        case UICollectionView.elementKindSectionHeader:
+            guard
+                let gradeView = collectionView.dequeueReusableSupplementaryView(ofKind: kind, withReuseIdentifier: "\(GradeCircleReusableView.self)", for: indexPath) as? GradeCircleReusableView,
```

Comment:
we have a Core extension for this: `collectionView.dequeue(GradeCircleReusableView.self, ofKind: kind, for: indexPath)`

That extension could also be improved to look more like the table view one. `let gradeView: GradeCircleReusableView  = collectionView.dequeue(ofKind: kind, for: indexPath)`